Clover/app/src/main/java/org/floens/chan/ui/controller/ImageViewerController.java

```diff
@@ -250,7 +251,9 @@ private void saveShare(boolean share, PostImage postImage) {
             ImageSaveTask task = new ImageSaveTask(postImage);
             task.setShare(share);
             if (ChanSettings.saveBoardFolder.get()) {
-                task.setSubFolder(presenter.getLoadable().boardCode);
+                task.setSubFolder(presenter.getLoadable().site.name() +
+                        File.separator +
+                        presenter.getLoadable().boardCode);
```

Comment:
I didn't make the site specific folder a separate option. So if you save in board specific folders, it will also always be under a board. Wasn't sure if it should be it's own option or not

Looking at this code now, my PR will put images under `site/board/image.jpg`, does it make more sense to put in the image in `board/site/image.jpg`?

Reply:
I think that site/board is fine.